### PR TITLE
refactor: add output format option in material_mapping.py

### PR DESCRIPTION
**Pull request by ShujieL**
add a command line option to specify map name and format in json or cbor

### CODE REVIEW

File: Examples/Scripts/Python/material_mapping.py
Comment (AJPfleger) [inline Examples/Scripts/Python/material_mapping.py:1]: I believe on l.111 you also need to change the format:
```
writeFormat=mapFormat,
```

File: Examples/Scripts/Python/material_mapping.py [hunk 1]
```diff
@@ -130,6 +132,27 @@ def runMaterialMapping(
...
 if "__main__" == __name__:
+
+    p = argparse.ArgumentParser(
+        description="Script to generate ACTS material map"
+    )
+    p.add_argument(
+        "-o","--outFile",
+        type=str,
+        default="material-map.json",
+        help="output filename for the generated material map. Now the json and cbor formats are supported",
```
Comment (AJPfleger) [inline Examples/Scripts/Python/material_mapping.py:143]:
```suggestion
        help="Output filename for the generated material map. Supported formats: JSON, CBOR.",
```

File: Examples/Scripts/Python/material_mapping.py [hunk 2]
```diff
@@ -139,4 +162,6 @@ def runMaterialMapping(
         outputDir=os.getcwd(),
         inputDir=os.getcwd(),
         readCachedSurfaceInformation=False,
+        mapName=mapName,
+        mapFormat=fmt
```
Comment (AJPfleger) [inline Examples/Scripts/Python/material_mapping.py:166]:
```suggestion
        mapName=mapName,
        mapFormat=mapFormat
```
Maybe, it makes sense to change from `fmt` to `mapFormat` to follow the overall style and be a bit more explicit.

**Review by AJPfleger — COMMENTED** [review]
To get rid of the python-formatting error in the CI:
https://acts.readthedocs.io/en/latest/contribution/run_formatting.html#python-formatting

**Comment by ShujieL** [comment]
> To get rid of the python-formatting error in the CI:
> https://acts.readthedocs.io/en/latest/contribution/run_formatting.html#python-formatting

Thanks. The code passed the check with black locally.

**Comment by AJPfleger** [comment]
That is odd. I get:
```
$ black material_mapping.py 
reformatted material_mapping.py

All done! ✨ 🍰 ✨
1 file reformatted.
```

Could you check again?

**Comment by ShujieL** [comment]
> That is odd. I get:
> 
> ```
> $ black material_mapping.py 
> reformatted material_mapping.py
> 
> All done! ✨ 🍰 ✨
> 1 file reformatted.
> ```
> 
> Could you check again?

OK I apparently didn't push the reformatted version. It should be good now.

**Review by AJPfleger — APPROVED** [review]
Thanks, for checking again. Looks good now!

**Comment by paulgessinger** [comment]
@andiwand @ShujieL Invite sent.

**Comment by ShujieL** [comment]
> @andiwand @ShujieL Invite sent.

@paulgessinger I didn't receive any notification. Can you resend?

**Comment by ShujieL** [comment]
@paulgessinger @andiwand just wanted to follow up. I didn't get any github notification on "CI bridge". Is that why this PR is stuck?

**Comment by andiwand** [comment]
@paulgessinger can we get `CI Bridge` permissions for @ShujieL ?

**Review by andiwand — APPROVED** [review]

**Comment by acts-project-service** [comment]
## :white_check_mark: Athena integration test results [360fc152b007fda7d48a3f74fa9f9d8431f51ee4]
### :white_check_mark: All tests successful

|status|job|report|
|------|---|------|
|:green_circle: | [run_unit_tests](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566186)||
|:green_circle: | [test_ActsCheckObjectCountsWorkflow](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566185)||
|:green_circle: | [test_ActsEFTrackFit](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566184)||
|:green_circle: | [test_ActsPersistifySeeds](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566183)||
|:green_circle: | [test_ActsBenchmarkWithSpot](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566182)||
|:green_circle: | [test_ActsAnalogueClustering](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566181)||
|:green_circle: | [test_ActsWorkflowHeavyIons](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566180)||
|:green_circle: | [test_ActsWorkflowFastTracking](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566179)||
|:green_circle: | [test_ActsWorkflowCached](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566178)||
|:green_circle: | [test_ActsWorkflow](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566177)||
|:green_circle: | [test_ActsValidateAmbiguityResolution](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566176)||
|:green_circle: | [test_ActsValidateResolvedTracks](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566175)||
|:green_circle: | [test_ActsValidateTracks](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566174)||
|:green_circle: | [test_ActsValidateActsCoreSpacePoints](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566173)||
|:green_circle: | [test_ActsValidateActsSpacePoints](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566172)||
|:green_circle: | [test_ActsValidateSeeds](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566171)||
|:green_circle: | [test_ActsValidateOrthogonalSeeds](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566170)||
|:green_circle: | [test_ActsValidateClusters](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566169)||
|:green_circle: | [test_ActsPersistifyEDM](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566168)||
|:green_circle: | [test_ActsGSFRefitting](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566167)||
|:green_circle: | [test_ActsKfRefitting](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566166)||
|:green_circle: | [test_ActsExtrapolationAlgTest](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566165)||
|:green_circle: | [test_ActsITkTest](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566164)||
|:green_circle: | [run_workflow_tests_run4_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566163)||
|:green_circle: | [run_workflow_tests_run2_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566162)||
|:green_circle: | [run_workflow_tests_run2_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566161)||
|:green_circle: | [run_workflow_tests_run3_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566160)||
|:green_circle: | [run_workflow_tests_run3_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566159)||
|:green_circle: | [run_art_test: test_data18_13TeV_1000evt](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566158)||
|:green_circle: | [run_art_test: test_ttbarPU40_reco](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/39566157)||

**Pull request merged**